inst/include/dust/dust.hpp

```diff
@@ -341,6 +435,10 @@ class Dust {
     return _n_pars;
   }
 
+  size_t n_pars_effective() const {
```

Comment:
`n_pars_size` might be a better name?

Reply:
Not sure. It's the number of "logical" parameter sets; in the case of a non-multi-pars object `n_pars_` is 0 but there are cases where want that number. There's no great name I suspect